daemon/indexing/apollo/index.js

```diff
@@ -246,7 +246,7 @@ const resolvers = {
       return { walletAddress: 'S_WADDR' }
     },
     title(listing) {
-      return listing.name
+      return listing.title || "Untitled"
```

Comment:
Would it make sense to actually return an empty string, so that the Dapp can differentiate a listing with no title vs with title ?

Now, note that the schema enforces a title is set since it is a "required" field: https://github.com/OriginProtocol/origin-js/blob/master/src/schemas/listing-core.json#L178
So this should never happen. But agreed better to be paranoid...